### DISCUSSION TITLE
0.13.0 Release

### DISCUSSION THREAD

**Discussion by gramakri** [Stirling-Tools/Stirling-PDF#326] [General]
There is a tag for 0.13.0 - https://github.com/Frooodle/Stirling-PDF/tags . But there is no Release. Maybe it's just taking time, but I thought I will alert you about it.

**Comment by Frooodle** [comment]
No sorry its testing tag
It's why it's not listed as a latest
I am going to re create today

**Comment by Frooodle** [comment]
Been making a few tags recently 
Also sorry your cloudron logic will need to change this release I am changing logic to produce two different jars which are selected based on env 

One for security and login features and one without

**Comment by Frooodle** [comment]
Done :)
https://github.com/Frooodle/Stirling-PDF/releases/tag/v0.13.0

let me know if you need any help on logic

**Comment by gramakri** [comment]
@Frooodle thanks! No worries with the package we are updating it to use the new yml config file

**Reply by Frooodle** [reply]
I see some people went down route of pure env config still like
https://mariushosting.com/how-to-install-stirling-pdf-on-your-synology-nas/
Just did and avoid using settings file
Which is also a option eg

environment: DOCKER_ENABLE_SECURITY: true # or false SECURITY_ENABLELOGIN: true #or false SECURITY_INITIALLOGIN_USERNAME: marius SECURITY_INITIALLOGIN_PASSWORD: mariushosting SECURITY_CSRFDISABLED: true #or false SYSTEM_DEFAULTLOCALE: en_US # or fr_FR or de_DE UI_APPNAME: mariusPDF UI_HOMEDESCRIPTION: mariushosting PDF Description UI_APPNAMENAVBAR: mariushosting PDF SYSTEM_MAXFILESIZE: 5000 # Set the maximum file size in MB METRICS_ENABLED: true SYSTEM_GOOGLEVISIBILITY: false # or true restart: on-failure:5
